mbedtls-sys/build/bindgen.rs

```diff
@@ -125,10 +129,25 @@ impl super::BuildConfig {
                 _ => {} // skip toolchains without a configured sysroot
             };
         }
+
+        // generate static function wrappers
+        let static_wrappers_c_path = self.out_dir.join("static_wrappers.c");
+        headers.push_str("#include <psa/crypto.h>\n");
+        bindgen::builder()
+            .enable_function_attribute_detection()
+            .clang_args(cc.get_compiler().args().iter().map(|arg| arg.to_str().unwrap()))
+            .header_contents("bindgen-input.h", &headers)
+            .wrap_static_fns(true)
+            .wrap_static_fns_path(&static_wrappers_c_path)
+            .generate().expect("bindgen error");
```

Comment:
Does this require a separate invocation?

Reply:
The main reason of separate invocation is: custom callback `MbedtlsParseCallbacks` affects C code generated by `wrap_static_fns`:
- results wrong rust binding code
- C code should not being affected by custom callback `MbedtlsParseCallbacks` which is for rust code generation

So the workflow here change to:

1. generate static function wrappers C code first
2. thenC `mbedtls` header and static function wrappers code are used together for generating rust bindings